src/frontend/src/components/organisations/search/index.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from 'react';
+import { Button, Input, Label } from 'nhsuk-react-components';
+
+const OrganisationsSearch: React.FC = () => {
+
+    const [name, setName] = useState('');
+    const [postCode, setPostCode] = useState('');
+    const [organisation, setOrganisation] = useState('');
+
+    const handleSearch = (e: React.MouseEvent<HTMLButtonElement>) => {
+        e.currentTarget.blur();
+    }
+
+    return (
+        <div data-testid='organisation-search'>
+            <Label isPageHeading={true} size='l'>Organisation search</Label>
+            <div className='nhsuk-u-margin-bottom-3' data-testid='search-by'>Search by <strong>either</strong> name, postcode or managing organisation.</div>
+
+            <Input
+                label="Name"
+                value={name}
+                onChange={(e: React.ChangeEvent<HTMLInputElement>) => setName(e.target.value)}
```

Comment:
can we move this into `const handleChange`

Reply:
Changed to use handleInputChange